### PR TITLE
Add disable option

### PR DESCRIPTION
**Pull request by ramybenaroya**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ module.exports = {
   shouldIncludeFiles: function() {
     var config = this.app.project.config()['ember-cli-mirage'];
 
-    return config.force || (this.app.env !== 'production')
+    return !config.disbale && (config.force || (this.app.env !== 'production'))
```
Comment (samselikoff) [inline index.js]: `disable` looks misspelled, are you sure the test is working correctly?

**Comment by ramybenaroya** [comment]
Sorry about this one. There were two mistakes: (1) typo (2) `disabled` was set to `false` in the config fixture.
Fixed

**Comment by ramybenaroya** [comment]
Wow, really sorry about this one. There were three mistakes: (1) "disbale" typo (2) disabled was set to false in the config fixture (3) I Didn't use the right config fixture for the test. 
Fixed.

**Comment by trabus** [comment]
@ramybenaroya I've currently got a [PR going here](https://github.com/samselikoff/ember-cli-mirage/pull/65) that achieves a similar result. The config is managed by an array containing the names of environments that mirage should be enabled in.

**Comment by ramybenaroya** [comment]
@trabus, Sam already merged [#72](https://github.com/samselikoff/ember-cli-mirage/pull/72). Check out the conversation. He prefer single "enabled" option though.

**Comment by trabus** [comment]
I did not see that! I'll rebase and go with your changes then.

**Pull request closed without merging**